packages/code/src/Code.tsx

```diff
@@ -140,8 +282,34 @@ function Code({
   chromeTitle = '',
   ...rest
 }: CodeProps) {
+  const scrollableMultiLine = useRef<HTMLPreElement>(null);
+  const scrollableSingleLine = useRef<HTMLDivElement>(null);
+  const [state, dispatch] = useReducer(scrollReducer, initialState);
```

Comment:
Hmm... I'm thinking useReducer might be a bit heavy-handed here. Thoughts @hswolff?

Reply:
Swapped back to useEffect now 👍